mage_ai/orchestration/db/__init__.py

```diff
@@ -71,8 +72,25 @@ def close_session(self):
         self.session = None
 
 
+def get_postgresql_schema(url):
+    parse_result = urlparse(url)
+    if parse_result.scheme == 'postgresql+psycopg2':
+        return parse_qs(parse_result.query)['options'][0].split('=')[1]
```

Comment:
Can we make sure the key is `search_path` the `options`? There could be other params used in options, right?

Reply:
Good catch: enhanced the parsing logics.